python3.10/hello-pt/cookiecutter.json

```diff
@@ -0,0 +1,13 @@
+{
+    "project_name": "sam-app-powertools",
+    "runtime": "python3.9",
```

Comment:
This should be `python3.10` right?

Reply:
Yup. I just missed this when going through.